snippets/perl/get_nb_catalog_images.pl

```diff
@@ -0,0 +1,46 @@
+#Load module netbackup.pm from current directory
+use lib '.';
+
+use netbackup;
+use Getopt::Std;
+
+my %args;
+getopt('supdt', \%args);
+my $fqdn_hostname = $args{s};
+my $username = $args{u};
+my $password = $args{p};
+my $domainname = $args{d};
+my $domaintype = $args{t};
+
+if (!$fqdn_hostname || !$username || !$password) {
+  print "\nUsage : perl get_nb_catalog_images.pl -s <master_server> -u <username> -p <password> [-d <domain_name>] [-t <domain_type>]\n\n";
+  die;
+}
+
+print "\nRecieved the following parameters : \n";
```

Comment:
It might help declutter the output if this type of information is printed under a verbose option.

Reply:
done